Makefile

```diff
@@ -42,9 +42,9 @@ venv: $(VIRTUAL_ENV)
 
 PIP_COMPILE_FLAGS := --allow-unsafe --generate-hashes $(PIP_COMPILE_OPTIONS)
 compile-deps: $(VIRTUAL_ENV)
-	pip-compile $(PIP_COMPILE_FLAGS) -o requirements/base.txt requirements/base.in
-	pip-compile $(PIP_COMPILE_FLAGS) -o requirements/test.txt requirements/test.in
-	pip-compile $(PIP_COMPILE_FLAGS) -o requirements/dev.txt requirements/dev.in
+	$(EXEC_CMD) pip-compile $(PIP_COMPILE_FLAGS) -o requirements/base.txt requirements/base.in
+	$(EXEC_CMD) pip-compile $(PIP_COMPILE_FLAGS) -o requirements/test.txt requirements/test.in
+	$(EXEC_CMD) pip-compile $(PIP_COMPILE_FLAGS) -o requirements/dev.txt requirements/dev.in
```

Comment:
KO chez moi, mais ça fonctionnait bien sans le EXEC_CMD. ¯\_(ツ)_/¯

```
python setup.py egg_info did not run successfully.
...
pip._internal.exceptions.MetadataGenerationFailed: metadata generation failed
```

Reply:
@francoisfreitag voici la stack complète après un rebuild : https://www.evernote.com/shard/s18/sh/bb2a8c6a-737c-4ff7-9a29-94b42ea03858/jNetMpJFM0EYwiy_wDVk69KWqKOoO18IEyEJkEDLRSF7vtQJ0DexvyhaYw

Reply:
Merci ! J’ai un souci pour y accéder, Evernote me dit :

> The URL provided does not correspond to a valid shared note. This may be caused by a typo in the link, or the note owner may have made it private.

Reply:
@francoisfreitag nouvel essai https://www.dropbox.com/s/3nys16spl61a5v2/bug_report_DELETEME.txt?dl=0

Reply:
Merci ! Ça a marché.
Ce qui nous intéresse ici est :

> Error: pg_config executable not found.

C’est corrigé par https://github.com/betagouv/itou/pull/2807/commits/ce04d058310b908cfef7035fce805f35177bec28.